USING_VANAGON.md

```diff
@@ -0,0 +1,42 @@
+# Using Vanagon
+- Note that these docs are incomplete. Docs for all parts of vanagon are on their way
+
+## Creating a Vanagon Project for Windows
+
+### Overview
+Vanagon projects for windows use [WiX](http://wixtoolset.org/) to create MSIs that will install on native windows platforms.  Vanagon uses the same process between all platforms, including windows, to build and stage artifacts before creating an installer.  Vanagon only uses WiX files for the actual installer itself, so the smaller / simpler your install experience, the less WiX you need. You can find example WiX files under [examples](https://github.com/puppetlabs/vanagon/examples/resources/windows/wix).  These files should provide a good start to create an MSI using vanagon.
+
+### Components
+The process vanagon uses to actually build the MSI doesn’t break the normal path used for Unix based projects; vanagon builds each component on a build system, moves everything to a staging directory, then packages everything up.  In order to construct components on a windows system follow the usual vanagon pattern set out in the [styleguide](https://github.com/puppetlabs/vanagon/blob/master/STYLEGUIDE.md).
+
```

Comment:
You also need to note that "Component" in the MSI relates to an individual installable component (e.g. file, service), whereas in Vanagon, it refers to the whole facility or sub-group within a package.